build.gradle

```diff
@@ -85,6 +85,7 @@ ext {
         junit4Dataprovider:     '2.6',
         mockito:                '3.3.3',
         powermock:              '2.0.7',
-        slf4jTest:              '2.1.1'
+        slf4jTest:              '2.1.1',
+        jackson:                '2.10.3'
```

Comment:
put this declaration between `httpclient` and `javaxInject` (the versions are grouped into implementation and test ones, the versions are sorted in alphabetical order in the groups)